src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -217,6 +239,7 @@ const NavHeader = (props) => {
             width: '100%'
           }}>
           <Search
+            getSearchLayout={setSearchHeight}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The prop `getSearchLayout` of the `Search` component has been changed from `setSearchText` to `setSearchHeight`. This seems like a significant change in functionality. Please ensure that this change is intentional and doesn't break the `Search` component's functionality.



<!-- This is an auto-generated comment by OSS CodeRabbit -->